### DISCUSSION TITLE
Run script without AdditionalSignaturePath

### DISCUSSION THREAD

**Discussion by Ben-munich** [Set-OutlookSignatures/Set-OutlookSignatures#70] [General]
Hello, 
is it possible to provide a value to the parameter "AdditionalSignaturePath" which will result in the script NOT creating an additional copy of the signatures somewhere else?

Thanks!

**Comment by GruberMarkus** [comment]
Hi @Ben-munich,

thank you for rising this issue.

The code has already been adopted to solve this problem: The next release will contain a fix that allows to pass an empty string for the "AdditionalSignaturePath" parameter. Than, the signatures will not be copied to an additional folder.

**Reply by Ben-munich** [reply]
Thanks for the Info! 👍
